test/model/workloads/WT-13252.workload

```diff
@@ -0,0 +1,20 @@
+create_table(1, table, Q, Q)
+
+begin_transaction(100)
+insert(1, 100, 1234, 1)
+commit_transaction(100, 100)
+
+set_stable_timestamp(200)
+checkpoint()
+
+begin_transaction(100)
+remove(1, 100, 1234)
+prepare_transaction(100, 300)
+evict(1, 1234)
```

Comment:
The `evict` operation is not supported by the version of the workload runner that is currently in the 8.0 branch. As such, it would probably make sense to exclude this file from the backport.

A separate question is whether it would make sense _at some point_ to backport the implementation of the `evict` operator, so that we would not need to exclude files from backports in the future. But that's for later.

Reply:
Thanks Peter, I'll remove this file.